pyproject.toml

```
@@ -79,8 +79,10 @@ dependencies = [
   "databricks-labs-pylint~=0.4.0",
   "databricks-labs-pytester>=0.3.0",
   "mypy~=1.10.0",
-  "numpy==1.26.4",
-  "pandas==1.4.1",
+  "numpy",
+  "pandas",
+    "pandas-stubs",
```

Nit: Let's try to keep it clean… ;)
```suggestion
  "pandas",
  "pandas-stubs",
```